distribution/tools/java-version-checker/src/main/java/org/elasticsearch/tools/java_version_checker/JavaVersionChecker.java

```diff
@@ -9,55 +9,18 @@
 package org.elasticsearch.tools.java_version_checker;
 
 import java.util.Arrays;
-import java.util.Locale;
 
 /**
- * Simple program that checks if the runtime Java version is at least 17.
+ * Java 17 compatible main which exits with an error.
```

Comment:
I find this confusing. The whole point of this is to _not_ exit with an error, right?

Reply:
Oops, this was a copy/paste problem.